specification/resources/resource-manager/readme.md

```diff
@@ -111,6 +101,8 @@ input-file:
 - Microsoft.Authorization/stable/2021-06-01/policyDefinitions.json
 - Microsoft.Authorization/stable/2021-06-01/policySetDefinitions.json
 - Microsoft.Authorization/preview/2020-07-01-preview/policyExemptions.json
+- Microsoft.Authorization/preview/2022-08-01-preview/policyVariables.json
```

Comment:
you should not add `preview`API to a `stable` tag.

Reply:
Otherwise PR LGTM

Reply:
@weidongxu-microsoft I checked back with a colleague on why we did this earlier with exemptions resource type, and apparently it was due to a billing related issue and we need to be able to change a billing charasteristic for these in the future. Same thing applies to variables resource type. 

Since this tag already has a -preview type before I'm guessing the sdks won't have any backcompat issues.

Reply:
lmk if that's ok by you, otherwise I can go back and see what else we can do.

Reply:
@tadelesh @msyyc @qiaozha @XiaofeiCao for awareness of an existing preview json in stable tag (and 2 new ones).

Ideally the 2021-06 tag should only contain the stable jsons, and another e.g. 2022-08-preview tag could contain all.
But given that `2020-07-01-preview/policyExemptions.json` already there, I don't have a good recommendation...

Reply:
@robga as fyi